### ISSUE TITLE
Hussian et al. "Dynamic neural network architecture inspired by the immune algorithm to predict preterm deliveries in pregnant women"

### ISSUE THREAD

**Issue by GillesVandewiele** [GillesVandewiele/EHG-Oversampling#23]
## Reported result

the  proposed approach  shows  an  improvement  on  existing  studies  with 89%sensitivity, 91%specificity, 90%positive  predicted  value, 90%negative  predicted  value,  and  an overall accuracy of 90%

## Features

root  mean  squares, peak frequency, median  frequency,  and sample  entropy.

## Model

The  self-organised  network  inspired  by the immune  algorithm  is  developed to improve  recognition  and  generalization  capability  of  the  backpropagation  neural networks. 

## Oversampling

The  first  evaluation  uses  the  original  TPEHG  dataset  (38 pretermand  262 term)–the  preterm  are  oversampled  using  min  and  max  to  produce  262 pretermrecords). 

--> Very unclear...

## Comments

Sounds like a modeling technique that never gets used...  But, other more well-known techniques are reported in the paper as well, so we can use one of these (Decision Trees, SVM).
